### ISSUE TITLE
[Suggestion] toString() method for neow3j.protocol.core.methods.response.NeoBlock class

### ISSUE THREAD

**Issue by cloud8little** [neow3j/neow3j#4]
```
 @Override
    public String toString() {
        return "NeoBlock{" +
                "hash='" + hash + '\'' +
                ", size=" + size +
                ", version=" + version +
                ", prevBlockHash='" + prevBlockHash + '\'' +
                ", merkleRootHash='" + merkleRootHash + '\'' +
                ", time=" + time +
                ", index=" + index +
                ", nonce='" + nonce + '\'' +
                ", nextConsensus='" + nextConsensus + '\'' +
                ", script=" + script +
                ", transactions=" + transactions +
                ", confirmations=" + confirmations +
                ", nextBlockHash='" + nextBlockHash + '\'' +
                '}';
```
I would suggest them to be showed with raw JSON format, what do you think?

**Comment by gsmachado** [comment]
@cloud8little I would keep the default notation when overriding the `toString()` method.

If you would like to have the JSON pretty-print notation, you could use the Jackson object mapper in your own code (`com.fasterxml.jackson.databind.ObjectMapper` class):

```
ObjectMapper mapper = new ObjectMapper();
String objString = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(blockReqResult.getBlock());
System.out.println(objString);
```

Then developers could print whatever format they would like to.

I will think about adding a `toJson()` method for objects representing JSON-RPC responses -- for convenience.

**Comment by gsmachado** [comment]
@cloud8little a fix was released on `1.0.7`.
If you experience that the issue was not yet fixed, please re-open it.

**Comment by cloud8little** [comment]
Use ObjectMapper class can be an option, it would be great that toJson() method for objects representing JSON-RPC responses to be implemented.
